### DISCUSSION TITLE
About trianing dataset

### DISCUSSION THREAD

**Discussion by tl32rodan** [google-research/perceptual-quality#3] [General]
Thanks for sharing the code! the works are inspiring. 
I'm interested about the training dataset you adopted. According to "An Unsupervised Information-Theoretic Perceptual Quality Metric", the dataset is public contains >40000 video sequences and more than 30 frames each. I wonder if there's any keyword to search for, or I can get access to that in any way. Thanks!

**Comment by jonaballe** [comment]
Hi tl32rodan, the dataset we trained with strictly consists of publicly accessible videos from YouTube, but the selection of videos is not static, since videos can be deleted or made private, etc., and then we remove them from the dataset as well. For compliance reasons, we cannot store the video data anywhere, so it is difficult to make them available. But it should not be hard for you to come up with a similar dataset. Essentially, we select random videos, only exclude certain categories such as artificial or screen content.
